### PR TITLE
Move Cask specs into `brew tests`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Homebrew/cask/cmd/brew-cask-tests.rb [hunk 1]
```diff
@@ -23,23 +23,5 @@ def run_tests(executable, files, args = [])
 
   if ARGV.flag?("--coverage")
     ENV["HOMEBREW_TESTS_COVERAGE"] = "1"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cask/cmd/brew-cask-tests.rb]: What does `brew cask-tests` do now?

Comment (reitermarkus) [inline Library/Homebrew/cask/cmd/brew-cask-tests.rb]: Nothing, but it is still needed because `test-bot` still calls it.

**Comment by reitermarkus** [comment]
Finally coverage is (*seems to be, at least* 😉 ) working.

**Comment by MikeMcQuaid** [comment]
@reitermarkus Nice work 👏

**Pull request merged**